frites/conn/tests/test_conn.py

```diff
@@ -155,6 +155,22 @@ def test_conn_pid(self):
             verbose=False, gcrn=True
         )
 
+    def test_conn_fit(self):
+        """Test function conn_fit."""
+        ar_type = 'hga'
+        n_stim = 2
+        n_epochs = 100
+        ss = StimSpecAR()
+        x = ss.fit(ar_type=ar_type, n_epochs=n_epochs, n_stim=n_stim)
+
+        for m in ['cc', 'cd']:  # mi_type
+            for n in [True, False]:  # net
+                for ad in [True, False]:  # avg_delay
+                    fit = conn_fit(x, y='trials', roi='roi', times='times',
+                                   mi_type=m, max_delay=.3, net=n,
+                                   verbose=False, avg_delay=ad)
+                    print(fit.shape)
```

Comment:
Here, can you put an assert to test that the shape of the fit is actually changing? Something like:

```python
if n:
    assert len(fit['roi']) == 1
else:
    assert len(fit['roi']) == 2

if ad:
    assert fit.ndim == 3
else:
    assert fit.ndim == 4
```

Reply:
"assert fit.ndim == 4" fails (it is 2).

Reply:
You're right, should be : 
```python
if ad:
    assert fit.ndim == 2
else:
    assert fit.ndim == 3
```